README.md

```diff
@@ -1 +1,28 @@
 # Weather-App
+
+This project is one of the homeworks for the Code Your Future coding training course. This application is created to let user check the temperature of the city of their desire. It takes users input and fetch data accordingly from the server and displays the fetched data in the browser in a way that is easy to read to the users.
+
+
+# Team members
+
+#Prabesh
+
+Prabesh focuses into how to how to fetch data from weather Api that can be then used to display into the applications. Prabesh is also constantly in touch with other team members and suggest any suggesting to inprove the app's performance.
+
+#Louis
+
+Louis focuses into designing and creating visually pleasing UI/UX that is easy interact with. Louis is also constantly in touch with other team members and suggest any suggesting to inprove the app's performance and the implementation of the fetched data.
+
+#Lukas
```

Comment:
my name is `Lukasz` with a `z` at the end

Reply:
sorry for the missing z Lukasz

Reply:
np